src/components/Tile/tile.scss

```diff
@@ -0,0 +1,278 @@
+//===================
+//SM TILE STYLES
+//===================
+.tile--sm {
+  display: flex;
+  flex-direction: column;
+
+  & > .tile--text-container {
+    padding-left: 0;
+
+    & > p {
+      font-size: calc(14px + 2 * ((100vw - 320px) / 1264));
+    }
+  }
+
+  @include max-breakpoint('lg'){
+    margin-bottom: rem(20px);
+  }
+}
+
+.img--sm {
+  width: 100%;
+  height: auto;
+}
+
+.tile--sm a.bx--tile {
+  width: 100%;
+}
+
+.tile--optional .tile__info > p {
+  font-size: rem(14px);
+  line-height: rem(18px);
+  color: $ibm-colors__gray--60;
+}
+
+.tile--optional .tile__info > h5:before {
+  height: 2.25rem;
+}
+
+.tile--optional .tile__info h5 {
+  max-width: rem(190px);
+  @media (max-width: 500px){
```

Comment:
for some reason the mixin wasn't working on this. Didn't give me an error or anything, it just wasn't doing anything  🤔